ui/app/templates/components/app-breadcrumbs.hbs

```diff
@@ -1,7 +1,7 @@
 <Breadcrumbs as |breadcrumbs|>
-  {{#each breadcrumbs as |crumb|}}
+  {{#each breadcrumbs as |crumb iter|}}
     {{#let crumb.args.crumb as |c|}}
-      {{component (concat "breadcrumbs/" (or c.type "default")) crumb=c}}
+      {{component (concat "breadcrumbs/" (or c.type "default")) crumb=c isOneCrumbUp=(eq iter (sub breadcrumbs.length 2))}}
```

Comment:
Since the active page is always the last breadcrumb, we want the penultimate one. Hence, `(eq iter (sub breadcrumbs.length 2)`